lib/cartopy/tests/mpl/test_examples.py

```diff
@@ -31,7 +32,9 @@ def test_global_map():
 
 
 @pytest.mark.natural_earth
-@pytest.mark.mpl_image_compare(filename='contour_label.png', tolerance=0.5)
+@pytest.mark.mpl_image_compare(
+    filename='contour_label.png',
+    tolerance=3.9 if MPL_VERSION.release[:2] < (3, 8) else 0.5)
```

Comment:
These images look really similar to me... I'd probably reverse this and leave the image out and just boost the tolerance to 3.9 for MPL3.8+. But, this seems fine as well.

Reply:
OK I reverted the test image.  In Iris we use a perceptual image hash, so I don't have the experience with RMS to know what counts as a big number.

Reply:
I'm not sure there is a good understanding of what "big number" is here either :) I was just basing it off of a quick look at the images. I think it will be quite a bit greater than 10 if there is a big difference (i.e. not just font offsets)

I thought @bjlittle thought about adding those hashes to Cartopy at some point, but I can't find that issue/PR... Maybe it was upstream in pytest-mpl instead https://github.com/matplotlib/pytest-mpl/pull/150
If that PR lands, it would probably be easier to add here as well I'd guess since we are using pytest-mpl already.